samples/net/ieee802154/hw/prj_fsci.conf

```diff
@@ -0,0 +1,50 @@
+CONFIG_NETWORKING=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+CONFIG_NET_PKT_RX_COUNT=10
+CONFIG_NET_PKT_TX_COUNT=10
+CONFIG_NET_BUF_RX_COUNT=20
+CONFIG_NET_BUF_TX_COUNT=20
+CONFIG_NET_MAX_CONTEXTS=10
+
+CONFIG_NET_IPV4=n
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV6_RA_RDNSS=y
+CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=2
+
+CONFIG_NET_6LO=y
+CONFIG_NET_DEBUG_6LO=y
+
+CONFIG_NET_LOG=y
+CONFIG_SYS_LOG_NET_LEVEL=1
+CONFIG_SYS_LOG_SHOW_COLOR=y
+CONFIG_INIT_STACKS=y
+CONFIG_PRINTK=y
+
+CONFIG_NET_SHELL=y
+
+CONFIG_NET_DEBUG_CORE=y
+CONFIG_NET_DEBUG_IPV6_NBR_CACHE=y
+CONFIG_NET_DEBUG_IPV6=y
+CONFIG_NET_DEBUG_CONTEXT=y
+CONFIG_NET_DEBUG_NET_PKT=y
+CONFIG_NET_DEBUG_UTILS=y
+CONFIG_NET_DEBUG_IF=y
+CONFIG_NET_DEBUG_ICMPV6=y
+CONFIG_NET_DEBUG_CONN=y
+CONFIG_NET_STATISTICS=y
+CONFIG_NET_STATISTICS_PERIODIC_OUTPUT=n
+
+CONFIG_NET_L2_IEEE802154=y
+CONFIG_NET_DEBUG_L2_IEEE802154=y
+CONFIG_NET_L2_IEEE802154_FRAGMENT=y
+#CONFIG_NET_DEBUG_L2_IEEE802154_FRAGMENT=y
+CONFIG_NET_L2_IEEE802154_SECURITY=n
+CONFIG_NET_L2_IEEE802154_SHELL=y
+
+CONFIG_IEEE802154_FSCI=y
```

Comment:
Note that PR #957 makes some changes how the prj files are used. So you might need to tweak these prj files a bit.